src/test/regress/expected/multi_partitioning.out

```diff
@@ -3763,6 +3763,152 @@ BEGIN;
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_16 | 01-16-2021 | 01-17-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_17 | 01-17-2021 | 01-18-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_18 | 01-18-2021 | 01-19-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_19 | 01-19-2021 | 01-20-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_20 | 01-20-2021 | 01-21-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_21 | 01-21-2021 | 01-22-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_22 | 01-22-2021 | 01-23-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_23 | 01-23-2021 | 01-24-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_24 | 01-24-2021 | 01-25-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_25 | 01-25-2021 | 01-26-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_26 | 01-26-2021 | 01-27-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_27 | 01-27-2021 | 01-28-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_28 | 01-28-2021 | 01-29-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_29 | 01-29-2021 | 01-30-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_30 | 01-30-2021 | 01-31-2021 | heap
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2021_01_31 | 01-31-2021 | 02-01-2021 | heap
+(31 rows)
+
+ROLLBACK;
+BEGIN;
+  SELECT create_time_partitions('date_partitioned_citus_local_table', INTERVAL '1 week', '2022-01-01', '2021-01-01');
+ create_time_partitions
+---------------------------------------------------------------------
+ t
+(1 row)
+
+  SELECT * FROM time_partitions WHERE parent_table = 'date_partitioned_citus_local_table'::regclass ORDER BY 3;
+            parent_table            | partition_column |                  partition                  | from_value |  to_value  | access_method
+---------------------------------------------------------------------
+ date_partitioned_citus_local_table | eventdate        | date_partitioned_citus_local_table_p2020w53 | 12-28-2020 | 01-04-2021 | heap
```

Comment:
I'm not sure but do you think does it add much value to print the all such records from time_partitionns individually ? Or should we instead execute a count(*)  query and rely on the resulting count ?